perl-file-rsyncp/riscv64.patch

```diff
@@ -0,0 +1,12 @@
+diff --git PKGBUILD PKGBUILD
+index a5c6bcc..0a0efd6 100644
+--- PKGBUILD
++++ PKGBUILD
+@@ -18,6 +18,7 @@ sha512sums=('b81d9ce63181605939e8aff87c9e56ab276899f019f44eb3f149e7ee0d44a94fcbe
+ prepare() {
+   cd File-RsyncP-$pkgver
+   # https://rt.cpan.org/Public/Bug/Display.html?id=141822
```

Comment:
This comment is about the gcc12.patch below, please don't insert an irrelevant fix in between :)